File: Program Kasir/kasir.java
import java.util.Scanner;

public class kasir {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        String nama;
        int jumlah;
        float harga, diskon, totalBayar, total;

        System.out.print("Masukan Nama Barang : ");
        nama = input.next();   

        System.out.print("Masukan Harga Barang : ");
        harga = input.nextFloat();

        System.out.print("Masukan Jumlah Barang : ");
        jumlah = input.nextInt();

        total = jumlah * harga;

        if (total > 85700)
            diskon = total * 0.07f;
        else
            diskon = 0;

        totalBayar = total - diskon;

        System.out.println("===============================");
        System.out.println("Nama Barang   : " + nama);
        System.out.println("Harga Barang  : " + harga);
        System.out.println("Jumlah Barang : " + jumlah);
        System.out.println("Total Harga   : " + total);
        System.out.println("Total Diskon  : " + diskon);
        System.out.println("Total Bayar   : " + totalBayar);

        input.close();
    }
}

File: Program Kasir/kasir.py
nama = input ("Masukan Nama Barang : ")
harga = float(input("Masukan Harga Barang : "))
jumlah = int(input("Masukan Jumlah Barang : "))

total = harga * jumlah

if total > 85700:
    diskon = total * 0.07
else:
    diskon = 0

    totalBayar = total - diskon

print("\n=========STRUK PEMBELIAN=========") 
print(f"Nama Barang   : {nama}")
print(f"Harga Barang  : Rp. {harga}")
print(f"Jumlah Barang : {jumlah}")
print(f"Total Harga   : Rp. {total}")
print(f"Diskon        : Rp. {diskon}")
print(f"Total Bayar   : Rp. {totalBayar}")
print("=================================")
    